src/index.js

```diff
@@ -3,5 +3,6 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import 'bootstrap/dist/css/bootstrap.css';
+import { BrowserRouter as Router } from 'react-router-dom';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<Router><App /></Router>, document.getElementById('root'));
```

Comment:
You generally want your ReactDom.render to look like so

ReactDOM.render(
<Router>
  <App />
</Router>, document.getElementById('root'));